### DISCUSSION TITLE
[Feature] Ability to filter out photos that don't have any geo data attached

### DISCUSSION THREAD

**Discussion by BernardoGiordano** [immich-app/immich#7619] [Feature Request]
### The feature

It would be useful to be able to filter out from the Photos page all the pictures without geo data attached. This way, I would feel more confident to bulk edit locations of my photos without overriding any existing data that may be present on some picture but not on others.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #5536
